### PR TITLE
Add known front to DST + add dst-concave

### PR DESCRIPTION
**Pull request by ffelten**
FYI, Just adding the known optimal fronts to the DST. I don't think we know the front of other environments though?

### CODE REVIEW

File: mo_gymnasium/envs/deep_sea_treasure/deep_sea_treasure.py [hunk 1]
```diff
@@ -25,6 +25,19 @@
...
+    np.array([11.5, -5]),
+    np.array([14.0, -7]),
+    np.array([15.1, -8]),
+    np.array([16.1, -9]),
+    np.array([19.6, -13]),
+    np.array([20.3, -14]),
+    np.array([22.4, -17]),
+    np.array([23.7, -19]),
+]
+
```
Comment (LucasAlegre) [inline mo_gymnasium/envs/deep_sea_treasure/deep_sea_treasure.py:40]: These fronts are for gamma = 1. Should we have separate fronts for different gammas? I think for DST people generally use 0.99

Comment (LucasAlegre) [inline mo_gymnasium/envs/deep_sea_treasure/deep_sea_treasure.py:40]: I don't you if I find better that these are global variables, or if they were returned by env.get_front() or something like that. What do you think?

Comment (LucasAlegre) [inline mo_gymnasium/envs/deep_sea_treasure/deep_sea_treasure.py:40]: These fronts are for gamma = 1. Should we have separate fronts for different gammas? I think for DST people generally use 0.99

Comment (LucasAlegre) [inline mo_gymnasium/envs/deep_sea_treasure/deep_sea_treasure.py:40]: I don't you if I find better that these are global variables, or if they were returned by env.get_front() or something like that. What do you think?

**Review by LucasAlegre — CHANGES_REQUESTED** [review]

**Comment by ffelten** [comment]
> I can easily compute the fronts of Resource Gathering. Minecart has a method that returns a CCS, but I'm not sure it is 100% accurate. It is a good approximation tho.

Could be cool to add these as well.

**Pull request merged**